test/framework/src/main/java/org/elasticsearch/telemetry/Measurement.java

```diff
@@ -54,7 +54,9 @@ public static List<Measurement> combine(List<Measurement> measurements) {
             }
             byAttr.compute(
                 m.attributes,
-                (k, v) -> (v == null) ? m.value : isDouble ? v.doubleValue() + m.getDouble() : v.longValue() + m.getLong()
+                (k, v) -> (v == null) ? m.value
```

Comment:
Can you turn this into a multi line lambda, so a normal if/else can be used? More than one level of ternary is incredibly difficult to visualize.

Reply:
Sure, absolutely right 👍